packages/components/src/presentation-components/properties/NavigationPropertyTargetSelector.scss

```
@@ -1,12 +1,9 @@
-.presentation-navigation-property-select-input {
-  > input,
-  > input:focus-visible {
-    width: 100%;
-    height: 100%;
-    border: none;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    outline: none;
-    padding: 0;
-  }
+.iui-input-with-icon.presentation-navigation-property-select-control .iui-select-button.presentation-navigation-property-select-input > input {
```

Would it be enough to specify only `presentation` related css classes?

```suggestion
.presentation-navigation-property-select-control .presentation-navigation-property-select-input > input {
```

---

not really, itwinui selector's specificity is (0,4,0), moved `border: none` and `outline: none` to inline